source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageRenderer.java

```diff
@@ -214,7 +212,8 @@ private static ConstraintSet createConstraints(Context context, ImageView imageV
         ConstraintSet constraints = new ConstraintSet();
 
         // ConstraintSet requires unique id
-        if(imageView.getId() == View.NO_ID) {
+        if(imageView.getId() == View.NO_ID)
```

Comment:
nitpick (since you fixed the same thing below):

```suggestion
        if (imageView.getId() == View.NO_ID)
```